packages/flutter/test/material/scrollbar_test.dart

```diff
@@ -556,4 +556,54 @@ void main() {
 
     await tester.pumpAndSettle();
   });
+
+  testWidgets("Scrollbar doesn't show when scroll the inner scrollable widget", (WidgetTester tester) async {
+    final GlobalKey outerKey = GlobalKey();
+    final GlobalKey innerKey = GlobalKey();
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: Scrollbar(
+            child: SingleChildScrollView(
+              key: outerKey,
+              child: SizedBox(
+                height: 1000.0,
+                width: double.infinity,
+                child: Column(
+                  children: <Widget>[
+                    Scrollbar(
+                      child: SizedBox(
+                        height: 300.0,
+                        width: double.infinity,
+                        child: SingleChildScrollView(
+                          key: innerKey,
+                          child: const SizedBox(
+                            key: Key('Inner scrollable'),
+                            height: 1000.0,
+                            child: ColoredBox(
+                              color: Colors.pink,
+                            ),
+                          ),
+                        ),
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+
+    // Drag the outer scrollable widget.
+    await tester.drag(find.byKey(outerKey), const Offset(0.0, -10.0));
+    await tester.pump();
+    await tester.pump(const Duration(milliseconds: 200));
+
+    // The outer scrollbar paint.
+    expect(find.byType(Scrollbar).first, paints..rect());
+
+    // Not finish.
```

Comment:
Need help!!!
This test case can run at the device and test the behavior normally, 
But I don't know how to trigger that drag the inner scrollable widget here.
I tried using `startGesture`, `moveBy`, and `drag` with the right offset, but all of them not working, and can not scroll the inner widget.
Can someone help me?